.circleci/scripts/check_mmi_trigger.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -eo pipefail
+
+# Ensure required environment variables are set
+if [ -z "$CIRCLE_PULL_REQUEST" ] || [ -z "$GITHUB_TOKEN" ]; then
+  echo "Error: CIRCLE_PULL_REQUEST and GITHUB_TOKEN must be set."
+  exit 1
+fi
+
+# Extract PR number from the pull request URL
+PR_NUMBER=$(echo "$CIRCLE_PULL_REQUEST" | awk -F'/' '{print $NF}')
+
+# Define repository details
+REPO_OWNER="$CIRCLE_PROJECT_USERNAME"
+REPO_NAME=$(basename "$CIRCLE_REPOSITORY_URL" .git)
+
+# Fetch PR details using GitHub API
+PR_DETAILS=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
```

Comment:
This token will not exist on PRs from forks. We should make sure this doesn't break CI for PRs from forks

Reply:
Oh but it's not yet satisfactory because an exit code of 1 is returned if the token is missing. Since this is an expected scenario that is not an error, it should not cause failure.

I'd recommend updating this to always skip MMI tests for forks.